src/SongManager.cpp

```diff
@@ -845,6 +845,43 @@ RString SongManager::SongToPreferredSortSectionName( const Song *pSong ) const
 	return RString();
 }
 
+
+void SongManager::GetPreferredSortSongsBySectionName( const RString &sSectionName, std::vector<Song*> &AddTo ) const
+{
+	for (PreferredSortSection const &v : m_vPreferredSongSort)
+	{
+		if (v.sName == sSectionName)
+		{
+			AddTo.insert( AddTo.end(), v.vpSongs.begin(), v.vpSongs.end() );
+			return;
+		}
+	}
+}
+
+std::vector<Song*> SongManager::GetPreferredSortSongsBySectionName( const RString &sSectionName ) const
+{
+	std::vector<Song*> AddTo;
+	for (PreferredSortSection const &v : m_vPreferredSongSort)

```

Comment:
Instead of duplicating the code, can we just use the above function?

`GetPreferredSortSongsBySectionName(sSectionName, AddTo)`

But also, why do we need two functions that essentially to the same thing?

Reply:
I _think_ i added two because I noticed similar functions also had two variants and wanted to stay consistent with that.